app/src/main/java/com/cliqz/jsengine/JSBridge.java

```diff
@@ -53,7 +53,7 @@
     private final Map<String, List<WaitingAction>> awaitingRegistration =
             new HashMap<>();
 
-    private static final long ACTION_TIMEOUT = 200;
+    private static final long ACTION_TIMEOUT = 500;
```

Comment:
What's the reason for the change? When the JS run loop is overloaded this will slow down page loading, as each request will take 500ms instead of 200.